File: libcommon/src/main/java/com/wang/mytest/common/util/ToastUtils.java
package com.wang.mytest.common.util;

import android.content.Context;
import android.widget.Toast;

public final class ToastUtils {

    private ToastUtils() {
    }

    public static void showShort(Context context, String text) {
        Toast.makeText(context, text, Toast.LENGTH_SHORT).show();
    }
}

File: libcommon/src/main/java/com/wang/mytest/common/util/ToastUtils.kt
package com.wang.mytest.common.util

import android.widget.Toast
import com.wang.mytest.common.util.AppUtils

object ToastUtils {

    fun showShort(text: String) {
        Toast.makeText(AppUtils.getApp(), text, Toast.LENGTH_SHORT).show()
    }
}
